chcemvediet/apps/anonymization/models.py

```diff
@@ -121,3 +211,19 @@ def datachecks(superficial, autofix):
     return itertools.chain(attachment_file_check(attachment_normalizations),
                            attachment_orphaned_file_check(attachment_normalizations, field,
                                                           AttachmentNormalization))
+
+@datacheck.register
+def datachecks_AttachmentRecognition(superficial, autofix):
+    u"""
+    Checks that every ``AttachmentRecognition`` instance, which file is not NULL, has its file
+    working, and there are not any orphaned attachment_recognition files.
+    """
+    # This check is a bit slow. We skip it if running from cron or the user asked for superficial
+    # tests only.
+    if superficial:
+        return []
+    attachment_recognitions = AttachmentRecognition.objects.all()
+    field = AttachmentRecognition._meta.get_field(u'file')
+    return itertools.chain(attachment_file_check(attachment_recognitions),
+                           attachment_orphaned_file_check(attachment_recognitions, field,
+                                                          AttachmentRecognition))
```

Comment:
Nebolo by to citatelnejsie zalomit takto?
```
    return itertools.chain(
        attachment_file_check(attachment_recognitions),
        attachment_orphaned_file_check(attachment_recognitions, field, AttachmentRecognition),
    )
```
Analogicky na ostatnych miestach, kde mame `itertools.chain`.

Reply:
:ok: